### DISCUSSION TITLE
PhDOS calculation

### DISCUSSION THREAD

**Discussion by Santoshkumarkaushik** [nguyen-group/QERaman#9] [General]
I want to calculate PhDOS using the output file generated by QERaman. When I run the q2r.x file ,it shows error that dyne0 file is not found. Actually it was not generated. How to resolve this sir? One more thing how to decide gamma and raman_gamma, nrs in raman.in file sir? Plz guide me in this regard.
![Uploading 17063600105041786106712144834543.jpg…]()

**Comment by nguyen-group** [comment]
Hi, the PhDOS can not be obtained from the QERaman since the QERaman only calculates the phonon at the Gamma point. 
For the phDOS, please check https://github.com/nguyen-group/QE-SSP/tree/master/gr/phdos

gamma and raman_gamma in raman.in are related to the lifetime of the photoexcited carrier and phonon, respectively. It can be obtained by fitting with the experimental data. Please find more information in our papers: 
https://www.sciencedirect.com/science/article/abs/pii/S0010465523003120
and https://journals.aps.org/prb/abstract/10.1103/PhysRevB.97.115407

nrs in raman.in is the number of calculated points of Raman shift. So the default value (500) is fine for most cases.

Best,
Nguyen

**Comment by Santoshkumarkaushik** [comment]
Thanks sir
